muckrock/core/views.py

```diff
@@ -734,3 +735,19 @@ def user_export(request):
         writer.writerow([user.username, user.profile.full_name, user.email])
 
     return response
+
+
+def dismiss_banner(request):
+    """Remember dismissed banners in session"""
+    if request.method == "POST":
+        banner_hash = request.POST.get("banner_hash")
+        if banner_hash:
+            dismissed_banners = request.session.get("dismissed_banners", [])
```

Comment:
Can we just store the last dismissed hash?  If it doesn't match, show the message.  No need to remember all hashes they have ever dismissed.